specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2022-10-01-preview/machineLearningServices.json

```diff
@@ -5336,6 +5385,49 @@
         "modelAsString": true
       }
     },
+    "Recurrence": {
+      "type": "object",
+      "description": "The workflow trigger recurrence for ComputeStartStop schedule type.",
+      "properties": {
+        "frequency": {
+          "$ref": "./mfe.json#/definitions/RecurrenceFrequency"
+        },
+        "interval": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The interval."
+        },
+        "startTime": {
+          "type": "string",
+          "description": "The start time."
+        },
+        "timeZone": {
+          "type": "string",
+          "description": "The time zone."
+        },
+        "schedule": {
+          "$ref": "./mfe.json#/definitions/RecurrenceSchedule"
+        }
+      }
+    },
+    "Cron": {
```

Comment:
why are you redefining an almost-exact-same-but-not definition as MFE's CronTrigger? why not just use the shared one, that way AML doesn't have two nearly-identical definitions for the same concept

Reply:
The sdk layer already has the concept of triggers merged. Merging the trigger concept at API level is a breaking change and we are not going ahead with that at this point.